controllers/statusreport/statusreport_adapter.go

```diff
@@ -35,8 +34,6 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const FeatureFlagStatusReprotingEnabled = "FEATURE_STATUS_REPORTING_ENABLED"
```

Comment:
@hongweiliu17 @MartinBasti wdyt, do we want to still keep this feature flag?

Reply:
no, we don't need it anymore

Reply:
I feel we don't need it any more either.

Reply:
Ok, thanks for clearing it up.